docs/csharp/language-reference/builtin-types/struct.md

```diff
@@ -0,0 +1,56 @@
+
+[!code-csharp[struct example](~/samples/csharp/language-reference/builtin-types/StructType.cs#StructExample)]
+
+Because a structure type is a value type, a variable of a structure type contains an instance of the type. That is called value semantics. By default, variables are copied on assignment, passing an argument to a method, or returning a method result. In the case of a structure-type variable, an instance of the type is copied (for the examples, see [Value types](value-types.md)). In high-performance scenarios that involve large structure types, you can avoid value copying and pass around an alias to a value-type variable. For information about how to do that, see [Write safe and efficient C# code](../../write-safe-efficient-code.md).
+
+Typically, you define a structure type when you need to represent a small data structure that has value semantics. For example, .NET uses structure types to represent a number (both [integer](integral-numeric-types.md) and [real](floating-point-numeric-types.md)), a [Boolean value](bool.md), a [Unicode character](char.md), a [time instance](xref:System.DateTime). If you need a type with reference semantics (that is, a variable of the type contains a reference to an instance of that type, not the instance itself), define a [class](../keywords/class.md).
+
+## Limitations with the design of a structure type
+
+When you design a structure type, you have the same capabilities as with a [class](../keywords/class.md) type, with the following exceptions:
+
+- You cannot declare a parameterless constructor. Every structure type already provides an implicit parameterless constructor that returns the [default value](default-values.md) of the type.
+
+- You cannot initialize an instance field at its declaration. However, you can initialize a [static](../keywords/static.md) or [const](../keywords/const.md) field at its declaration.
+
+- A structure type cannot inherit from other class or structure type and it cannot be the base of a class. However, a structure type can implement [interfaces](../keywords/interface.md).
+
+- You cannot declare a [finalizer](../../programming-guide/classes-and-structs/destructors.md) within a structure type.
+
+## Instantiation of a structure type
+
+You use the [`new`](../operators/new-operator.md) operator to instantiate a structure type. In the case of the [built-in value types](value-types.md#built-in-value-types), use the corresponding literals to specify a value of the type.
+
+You can also instantiate a structure type without the `new` operator. However, you must initialize all instance fields before the first use of the instance. The following example shows how to do that:
+
+[!code-csharp[without new](~/samples/csharp/language-reference/builtin-types/StructType.cs#WithoutNew)]
+
+If a structure type contains private fields, you must use the `new` operator to instantiate the type.
```

Comment:
I"m not sure what you're trying to explain here. I don't think its correct in that arrays of a struct type can be created (`new` isn't called on each element).

Reply:
Indeed, it's not correct; thank you for the array example. I'll remove that sentence.